fpdf/fpdf.py

```diff
@@ -2595,7 +2619,10 @@ def multi_cell(
             ]
         for text_line_index, text_line in enumerate(text_lines):
             is_last_line = text_line_index == len(text_lines) - 1
-
+            # fix issue #364
```

Comment:
The intent here is to uncomment that section in a later PR?

Reply:
Probably. I'll remove it for now.